.travis.yml

```diff
@@ -0,0 +1,26 @@
+branches:
+  only:
+  - /.*/
+dist: xenial
+language: python
+python:
+  - 3.6
+services:
+  - postgresql
+addons:
+  postgresql: "11"
+before_install:
+  - wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+  - sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -sc)-pgdg main" > /etc/apt/sources.list.d/PostgreSQL.list'
+  - sudo apt-get update
+  - sudo apt-get install -y postgresql-11
+  - sudo grep postgres /etc/postgresql/11/main/pg_hba.conf
```

Comment:
What is this and the identical line two lines below for?

Reply:
It generates output that ends up in the Travis build log, that shows the line in `pg_hba.conf` with the authentication details for the `postgres` user. The two lines aren't necessary for the build to succeed, but they were useful for debugging the `sed` command in the middle.